Tests/UnitTests/Plugins/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_subdirectory_if(Digitization ACTS_BUILD_DIGITIZATION_PLUGIN)
 add_subdirectory_if(Json ACTS_BUILD_JSON_PLUGIN)
 add_subdirectory_if(Cuda ACTS_BUILD_CUDA_PLUGIN)
+add_subdirectory_if(TGeo ACTS_BUILD_TGEO_PLUGIN)
```

Comment:
Nitpicking: could your order this alphabetically?

Reply:
Sure, will update.

Reply:
Done.